src/Controllers/ConfirmationController.php

```diff
@@ -89,6 +89,10 @@ public function showConfirmation(int $orderId = 0, $orderAccesskey = '')
                 ]
             );
         }
+        elseif(!$order instanceof LocalizedOrder && !is_null($order))
+        {
+            return $order;
```

Comment:
Was soll diese Änderung bewirken? Als Rückgabewert der Controller-Methode wird ein String erwartet. Hier wird ein unbekannter Datensatz zurückgegeben.

Reply:
Falls findOrderByAccessKey in die If-Abfrage für den redirect springt, dann muss dieser nach oben gereicht werden. Der redirect ist allerdings keine instance of Order sondern Response.. Deswegen ist diese Abfrage da, um den redirect der nicht vom typ order ist nach oben durchzugeben.